### ISSUE TITLE
If App is on React.StrictMode, Page doesnt re-render. 

### ISSUE THREAD

**Issue by albseb511** [Paratron/hookrouter#121]
If <React.StrictMode> is there on the top level, the page doesnt get rendered.

Have to hard reload it.

```
ReactDOM.render(
  <React.StrictMode>
      <App />
  </React.StrictMode>,
  document.getElementById('root')
);
```

**Comment by DangerN** [comment]
I also have this issue!

I have noticed that if you import `usePath` and invoke it in your `<App/>` component rendering behaves as expected. I haven't tested if this works only at the top level or at any level.

Anyone have any thoughts on where the issue might originate? I'll just go back to non-strict mode for now but I'm willing to help solve the problem.

**Comment by alete89** [comment]
+1

**Comment by zhaoyao91** [comment]
same problem here

**Comment by Paratron** [comment]
Fixed in 2.0. (coming in the next days)

Strict mode is really crazy in dev. It calls all internal lifecycle  methods twice, causing the router to be completely confused. But I got away with it.

**Comment by forresthopkinsa** [comment]
It would be nice to have a fix for this on v1. See #128, #155, probably others

**Comment by a613** [comment]
> Fixed in 2.0. (coming in the next days)

where's v2? more than a few days since May 2020
